test-app/src/main/java/com/mapbox/navigation/examples/core/MapboxJunctionActivity.kt

```diff
@@ -0,0 +1,260 @@
+            NavigationOptions.Builder(this)
+                .accessToken(getMapboxAccessTokenFromResources())
+                .locationEngine(ReplayLocationEngine(mapboxReplayer))
+                .build()
+        )
+        junctionApi = MapboxJunctionApi(getMapboxRouteAccessToken(this))
+        mapboxNavigation.startTripSession()
+        mapboxReplayer.pushRealLocation(this, 0.0)
+        mapboxReplayer.play()
+    }
+
+    @SuppressLint("MissingPermission")
+    private fun initStyle() {
+        mapboxMap.loadStyleUri(
+            MAPBOX_STREETS
+        ) {
+            getGesturePlugin().addOnMapLongClickListener(this)
+        }
+    }
+
+    private fun findRoute(origin: Point, destination: Point) {
+        mapboxNavigation.requestRoutes(
+            RouteOptions.builder()
+                .applyDefaultParams()
+                .accessToken(Objects.requireNonNull(getMapboxRouteAccessToken(this)))
+                .coordinates(listOf(origin, destination))
+                .alternatives(true)
+                .build()
+        )
+    }
+
+    @SuppressLint("MissingPermission")
+    private fun vibrate() {
+        val vibrator = getSystemService(VIBRATOR_SERVICE) as Vibrator
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            vibrator.vibrate(VibrationEffect.createOneShot(100L, DEFAULT_AMPLITUDE))
+        } else {
+            vibrator.vibrate(100L)
+        }
+    }
+
+    private fun getMapCamera(): CameraAnimationsPlugin {
+        return binding.mapView.getCameraAnimationsPlugin()
+    }
+
+    private fun getGesturePlugin(): GesturesPlugin {
+        return binding.mapView.getGesturesPlugin()
+    }
+
+    private fun getMapboxAccessTokenFromResources(): String? {
+        return getMapboxAccessToken(this)
+    }
+
+    private fun updateCamera(location: Location) {
+        val mapAnimationOptionsBuilder = MapAnimationOptions.Builder()
+        mapAnimationOptionsBuilder.duration(1500L)
+        mapCamera.easeTo(
+            CameraOptions.Builder()
+                .center(Point.fromLngLat(location.longitude, location.latitude))
+                .bearing(location.bearing.toDouble())
+                .pitch(45.0)
+                .zoom(17.0)
+                .padding(EdgeInsets(1000.0, 0.0, 0.0, 0.0))
+                .build(),
+            mapAnimationOptionsBuilder.build()
+        )
+    }
+
+    /**
+     * Uses a specific access token required for the route request to send junctions in the response.
+     * If the specific access token is not present, it defaults to access token required to render
+     * Maps.
+     *
+     * @param context The [Context] of the [android.app.Activity] or [android.app.Fragment].
+     * @return The Mapbox access token or null if not found.
+     */
+    private fun getMapboxRouteAccessToken(context: Context): String {
+        val tokenResId = context.resources
+            .getIdentifier("mapbox_route_token_zenrin", "string", context.packageName)
```

Comment:
Should we remove `zenrin` from here?

Reply:
I am not sure. We have different tokens for different features. Maybe I will change them to 
`mapbox_access_token_junction`, `mapbox_access_token_snapshot` and `mapbox_access_token_signboard`.
So, then we will have 4 tokens in total
`mapbox_access_token`
`mapbox_access_token_junction`
`mapbox_access_token_snapshot`
`mapbox_access_token_signboard`

cc @Guardiola31337